client/state/partner-portal/types.ts

```diff
@@ -72,6 +72,11 @@ export interface APILicense {
 	revoked_at: string | null;
 }
 
+export interface APILicenseAssign {
+	ok: boolean;
+	error: string;
+}
+
```

Comment:
This is unnecessary - the assign endpoint responds with a license which is covered by the `APILicense` interface.

Reply:
Good catch. Fixed in https://github.com/Automattic/wp-calypso/pull/60903/commits/9035c9ea6329895054bafb6d41f11392d381d966.